helpers/training/save_hooks.py

```diff
@@ -150,12 +169,6 @@ def _save_lora(self, models, weights, output_dir):
                 text_encoder_2_lora_layers_to_save = convert_state_dict_to_diffusers(
                     get_peft_model_state_dict(model)
                 )
-            # elif isinstance(
-            #     model, type(unwrap_model(self.accelerator, self.text_encoder_3))
-            # ):
-            #     text_encoder_3_lora_layers_to_save = convert_state_dict_to_diffusers(
-            #         get_peft_model_state_dict(model)
-            #     )
```

Comment:
text encoder 3 lora training is anyway not supported so, I have deleted `text_encoder_3` from the face of `save_hooks.py`. LMK if it should have been done differently.